### PR TITLE
add org.gnome.Aisleriot

### PR DESCRIPTION
**Pull request by nedrichards**

### CODE REVIEW

**Comment by nedrichards** [comment]
bot, build org.gnome.Aisleriot

File: org.gnome.Aisleriot.json [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            "sources": [
+            {
+                "type": "archive",
+                "url": "https://ftp.gnu.org/gnu/libunistring/libunistring-0.9.8.tar.xz",
+                "sha256": "7b9338cf52706facb2e18587dceda2fbc4a2a3519efa1e15a3f2a68193942f80"
+            }
+            ]
+        },
+        {
+            "name": "guile",
```
Comment (TingPing) [inline org.gnome.Aisleriot.json]: Probably installs binaries that can be cleaned up?

File: org.gnome.Aisleriot.json [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+            "name": "aisleriot",
+            "config-opts": [
+                "--with-card-theme-formats=svg"
+            ],
+            "sources": [
+                {
+                "type": "archive",
+                "url": "https://download.gnome.org/sources/aisleriot/3.22/aisleriot-3.22.4.tar.xz",
+                "sha256": "fe8dee3ad771ab778d37740a26410778aa5c61e8eb75dd42b9a5e5719c6e34fb"
+                }
```
Comment (TingPing) [inline org.gnome.Aisleriot.json]: Whitespace.

**Comment by nedrichards** [comment]
bot, build org.gnome.Aisleriot

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/org.gnome.Aisleriot

**Pull request closed without merging**